Mekanism_Client.launch

```diff
@@ -1 +1 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication"><stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStart"/><stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="Mekanism"/><stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="/Users/aidancbrady/Desktop/Minecraft Development/Projects/Mekanism/run"/><stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value=" -DFORGE_FORCE_FRAME_RECALC=true"/></launchConfiguration>
\ No newline at end of file
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><launchConfiguration type="org.eclipse.jdt.launching.localJavaApplication"><stringAttribute key="org.eclipse.jdt.launching.MAIN_TYPE" value="GradleStart"/><stringAttribute key="org.eclipse.jdt.launching.PROJECT_ATTR" value="Mekanism"/><stringAttribute key="org.eclipse.jdt.launching.WORKING_DIRECTORY" value="/home/joe/MinecraftMods/Mekanism/run"/><stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value=" -DFORGE_FORCE_FRAME_RECALC=true"/></launchConfiguration>
```

Comment:
can you please undo these `.launch` changes? The files really shouldn't be in git in the first place (I'll remove them on the main branch)

Reply:
Oops, done. (And +1 to removing them)